spec/order_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'order'
+require_relative '../lib/menu.rb' # need to stub
+
+
+describe Order do
+subject(:order_new){ described_class.new("jim","7","cam","nw10","+555") }
+  it "Can create a new Order that takes 5 arguments"\
+    "(name,house_no,street,postcode,phone_no)" do
+    expect(Order).to respond_to(:new).with(6).arguments
+  end
+
+  it "Can add and item and its quantity to the cart" do
+    menu = Menu.new
+    order = order_new
+    order.add_to_cart(menu,menu.itemised[0],2)
+    expect(order)
+      .to have_attributes(cart: [[["CHICKEN CACCIATORE","£13.95"], 2]] )
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver. (https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains)